src/components/common-components/modal/export-modal/export.modal.component.tsx

```diff
@@ -112,9 +112,10 @@ export default function ExportModal(options: ExportModalComponent): JSX.Element
       <div>
         <p className="label">Opcje pliku: </p>
         <FormGroup row>
-          {Object.keys(exportOptions).map((key) => (
+          {Object.keys(exportOptions).map((key, index) => (
             <FormControlLabel
               style={{ color: "black" }}
+              key={index}
```

Comment:
Index nie powinien być dodawany jako klucz. W tym przypadku możesz użyć klucza.  
https://robinpokorny.medium.com/index-as-a-key-is-an-anti-pattern-e0349aece318